src/pages/index.tsx

```diff
@@ -5,9 +5,9 @@ import { Banner, Carousel, Markdown } from '../components';
 import contentItemService from '../services/ContentItemService';
 
 const componentLookup = {
-  'https://get-started-dashboard.com/banner-carousel': Carousel,
-  'https://get-started-dashboard.com/banner-schema': Banner,
-  'https://get-started-dashboard.com/hello-world-schema': Markdown,
+  'https://schema-examples.com/banner-carousel': Carousel,
+  'https://https://schema-examples.com/tutorial-banner': Banner,
+  'https://get-started-dashboard.com/hello-world-schema': Markdown
```

Comment:
not a fan of the new inconsistency in schema uris. Why aren't they consistent?

Reply:
I still don't have access still but my question remains: why can't they consistently start with the same domain?